MANIFEST.in

```diff
@@ -1,3 +1,4 @@
 graft tests
 
 global-exclude *.py[co] .DS_Store
+prune tests
```

Comment:
Probably don't need to graft (above) and then prune them. If we usually include the tests, then let's just leave them in

Reply:
ahahah. So dump. Indeed !